src/games/Lexible/models/lexibleEndpoints.ts

```diff
@@ -0,0 +1,117 @@
+import MessageEndpoint from "libs/messaging/MessageEndpoint";
+import { Vector2 } from "libs";
+
+export interface PlayBoard {
+    gridWidth: number
+    gridHeight: number
+    gridData: string
+}
+
+// ------------------------------------------------------------------------------------------
+// LexiblePlayRequestMessage
+// ------------------------------------------------------------------------------------------
+export interface LexibleOnboardClientMessage {
+    roundNumber: number;
+    gameState: string;
+    playBoard: PlayBoard;
+    teamName: string;
+    settings: { startFromTeamArea: boolean }
+}
+
+export type LetterChain = {letter: string, coordinates: Vector2}[]
+
+export const LexibleOnboardClientEndpoint: MessageEndpoint<unknown, LexibleOnboardClientMessage> = {
```

Comment:
OK, so this is nice.  Endpoints are just light descripts of message interfaces and the logic to handle them lives in the client or presenter.  Cool.